tests/test_tax.py

```diff
@@ -745,19 +745,55 @@ def test_label_0(runtmp):
     assert "d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__Bacteroidales;f__Bacteroidaceae;g__Phocaeicola;s__Phocaeicola vulgatus" in lin_gather_results[3]
     assert "d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__Bacteroidales;f__Bacteroidaceae;g__Prevotella;s__Prevotella copri" in lin_gather_results[4]
 
-## some test ideas to start with -- see test_lca.py for add'l ideas
+def test_summarize_empty_gather_results(runtmp):
+    tax = utils.get_test_data('tax/test.taxonomy.csv')
+    
+    #creates empty gather result
+    g_csv = runtmp.output('g.csv')
+    with open(g_csv, "w") as fp:
+        fp.write("")
+    print("g_csv: ", g_csv)
+
+    #test below is not working properly: assert statement needs to be moved outside of the with block, but exc.value is empty and im not sure why
```

Comment:
```suggestion
```
this can go now, right? :)

Reply:
oh oops ok, this is the wrong version. Im on the github struggle bus.